### PR TITLE
Improve Tableau fragment

### PR DESCRIPTION
**Pull request by martin-dominguez**
I added some features to this fragment:

- Added configuration parameters instead of change the configuration directly in the HTML code. Added language properties for labels as well
- Separated JS from HTML
- Added JS code to work better during editing
- Updated docs
- Added parameters to use local or online private installations of Tableau

### CODE REVIEW

File: modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/tableau/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+var scriptElement = document.createElement("script");
+
+scriptElement.src = configuration.hostUrl + "/javascripts/api/viz_v1.js";
+
+vizElement.parentNode.insertBefore(scriptElement, vizElement);
+
+// Improving the behaviour while editing
+var editing = !!document.getElementsByClassName('fragments-editor').length;
+var editDiv = document.getElementsByClassName('edit')[0];
```
Comment (ethib137) [inline modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/tableau/index.js]: Hey @martin-dominguez, what does this do? I can't seem to find an element that has the className 'edit'.

Comment (martin-dominguez) [reply modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/tableau/index.js]: I've been taking a look at it, and it was included by Jan to show or hide a layer that AFAIK only showed a message.
`<div class="edit">edit mode</div>`
So, in this case, it seems to be unuseful. But I usually use this piece of code to better perform when editing, so, before change the code, let me make a question. @ethib137 should we avoid the load of the report when editing and just show the pre-load image?

I'd say that is useful see the report, to see its behavior with your page's design, but it's totally true that the performance is worst, at least during the report load.

In this last case, we should remove lines 14-21

Comment (ethib137) [reply modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/tableau/index.js]: I think it's helpful to load the report for now... If we run into a performance issue we can change it then. Thanks.

File: modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/tableau/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+if (editing && !!editDiv) {
+    editDiv.classList.remove("invisible");
+} else {
+    editDiv.classList.add("invisible");
+}
```
Comment (martin-dominguez) [inline modules/integrations-fragment-collection-contributor/src/main/resources/com/liferay/fragment/collection/contributor/integrations/dependencies/tableau/index.js]:
```suggestion

```

**Comment by ethib137** [comment]
Merged. Thanks.

**Pull request closed without merging**